ksql-functional-tests/src/main/java/io/confluent/ksql/test/serde/avro/ValueSpecAvroSerdeSupplier.java

```diff
@@ -164,6 +175,33 @@ private static Object valueSpecToAvro(final Object spec, final org.apache.avro.S
       }
     }
 
+    @SuppressWarnings("unchecked")
+    private static GenericRecord getAvroRecord(final Object spec, final Schema schema) {
+      final GenericRecord record = new GenericData.Record(schema);
+      final Map<String, String> caseInsensitiveFieldNames
+          = getCaseInsensitiveMap((Map) spec);
+      for (final org.apache.avro.Schema.Field field : schema.getFields()) {
+        record.put(
+            field.name(),
+            valueSpecToAvro(((Map<String, ?>) spec)
+                .get(caseInsensitiveFieldNames.get(field.name().toUpperCase())), field.schema())
+        );
+      }
+      return record;
+    }
+
+    @SuppressWarnings("unchecked")
+    private static GenericRecord getAvroRecordForMapEntry(
+        final Map.Entry<?, ?> spec,
+        final Schema schema) {
+      final GenericRecord record = new GenericData.Record(schema);
+      record.put("key",
```

Comment:
Added some comment along with a link.

Reply:
Link is great! Let's also use the constants `AvroData.KEY_FIELD` and `AvroData.VALUE_FIELD` to make it extra clear (actually in that case we probably don't even need the link because it'll be implied from the constant)

Reply:
Good point! I use the constants now!